tools/win/generate_breakpad_symbols.py

```diff
@@ -18,7 +18,7 @@
 import threading
 
 from datetime import datetime
```

Comment:
Looks like the file should be renamed to `generate_debug_symbols.py`. WDYT?

Reply:
wfm